srv/GetSimulatorFeatures.srv

```diff
@@ -0,0 +1,32 @@
+# Get a list of features supported by the simulator.
+# Simulators adhering to the standard need to implement at least this interface.
+
+---
+
+uint8 SPAWNING                  = 0      # Supports spawn interface (SpawnEntity).
```

Comment:
Isn't the usual test for this "does this node have the service interface"?

Reply:
I am not sure what you mean here, could you elaborate?

Reply:
In this scenario, let's pretend I'm a user who wants to spawn an entity in the simulator. 

When using this service, I would 
 * Call the service. 
 * See if GetSimulatorFeatures.Response.SPAWNING is present in response.features
 * If it is, then create a service client for `simulator/spawn_entity`

What I'm suggesting is that the test should just be
 * Check if `simulator/spawn_entity` exists as a service interface. 
 * If it is, then create the service client. 

Many of the other constants still work as retrievable features that you would want the service interface for (`SIMULATION_RESET_OBJECTS`), but I would leave out all that are `supports X interface`

Reply:
Apologies, I'm not understanding your perspective. In the lazy approach you describe, how do you get the information through a service at launch time?

Reply:
Consider a launcher which creates the feature service but not any other yet